src/main/java/com/wavedroid/musicbrainz/dao/MusicbrainzDao.java

```diff
@@ -132,7 +132,8 @@
             "         AS tbl) AS tbl2\n" +
             "ORDER BY artist, rg_year, rg_month\n";
 
-    private static final String RELEASE_BY_MBID = "SELECT\n" +
+    // delete, because it is redundant
```

Comment:
Why not just delete it? We're on git here

Reply:
okay